arm9/lib/libnns/src/NNS_FND_allocator.c

```diff
@@ -0,0 +1,32 @@
+#include "nitro.h"
+#include "NNS_FND_allocator.h"
+#include "NNS_FND_expheap.h"
+
+void *AllocatorAllocForExpHeap(NNSFndAllocator * pAllocator, u32 size)

```

Comment:
these funcs need ARM_FUNC

Reply:
i really want to move away from the ARM_FUNC/THUMB_FUNC standard in favor of just writing the thumb pragma in global.h